Leanplum-SDK/Classes/LeanplumRequest.m

```diff
@@ -417,7 +445,7 @@ - (void)sendRequests:(BOOL)async
             if (_error != nil) {
                 _error(err);
             }
-            [sendNowQueue cancelAllOperations];
+            [[LeanplumRequest sendNowQueue] cancelAllOperations];
```

Comment:
You can have multiple one if you call sendNow multiple times. Resume -> Pause -> getVars etc..